### ISSUE TITLE
Is Canvas still needed? 

### ISSUE THREAD

**Issue by diberry** [Azure/cosmos-explorer#414]
When running 'npm install', I get a sym link error for canvas. What installation instructions do you have for resolving this?

**Comment by southpolesteve** [comment]
@diberry On Windows or Mac? 

We don't use Canvas, but there is an unfortunate bug in an upstream version of nteract that requires we take a dependency on it. We mock it out to prevent having to actually install the canvas package, but its possible we did something in a non-cross platform way. https://github.com/Azure/cosmos-explorer/blob/master/canvas/README.md

**Comment by diberry** [comment]
It was Windows 10 corpnet version. I don't have very much globally installed in npm. What OS do you use?

**Comment by southpolesteve** [comment]
I'm on mac, but I have a windows machine too. Let me test it there and see if I can reproduce the issue

**Comment by diberry** [comment]
Is there a dockerfile that this repo runs in? Maybe that would be my quick fix.

**Comment by southpolesteve** [comment]
There is not. I have set it up once before in GitHub Codespaces if you want
to try that.

**Comment by diberry** [comment]
Works on my mac - lots of fun warnings and compilation errors - but it works. :)

**Comment by diberry** [comment]
How do I connect this local to my mongoDB via connection string?

**Comment by southpolesteve** [comment]
I don't think that will work. Mongo has to go through our portal proxy (since mongo doesn't support HTTP) which only works with production databases.
